src/QMCDrivers/DriftOperators.h

```diff
@@ -157,22 +139,45 @@ inline T setScaledDriftPbyPandNodeCorr(T tau,
  * @param qf quantum forces
  * @param drift scaled quantum forces
  * @param return correction term
+ *
+ * Fill the drift vector one particle at a time (pbyp).
+ *
+ * The naive drift is tau/mass*grad_psi_over_psi,
+ *  namely the log derivative of the guiding wavefunction; tau is timestep; mass is particle mass
+ * The naive drift diverges at a node, causing persistent configurations.
+ * The norm of the drift vector should be limited in two ways:
+ *  1. Umrigar: suppress drift divergence to mimic wf divergence
+ *  2. Ceperley: limit max drift rate to diffusion rate -> set Umrigar "a" parameter
+ * The choice of drift vector does not affect VMC correctness
+ *  so long as the proposal probabilities are correctly calculated.
+ * The choice of drift vector changes the DMC Green's function. BE CAREFUL!
+ *
+ * T should be either float or double
+ * T1 may be real or complex
```

Comment:
T1 may be real or complex of float or double.

Reply:
agreed. I will edit the comment.